docs/docsite/rst/os_guide/windows_winrm.rst

```diff
@@ -513,7 +513,7 @@ work. To troubleshoot Kerberos issues, ensure that:
   with ``nslookup``. The same name should be returned when using ``nslookup``
   on the IP address.
 
-* The Ansible host's clock is synchronized with the domain controller. Kerberos
+* The Ansible host's clock is synchronized with the domain control node. Kerberos
```

Comment:
@gotmax23 same question here - is this referring to a windows thing and not an ansible control node?